Validation/Geometry/src/MaterialBudgetCategorizer.cc

```diff
@@ -88,13 +88,13 @@ void MaterialBudgetCategorizer::buildHGCalCategoryMap(std::string theMaterialFil
     cms::Exception("LogicError") <<" File not found " << theMaterialFileName;
   
   // fill everything as "other"
-  float Air,Cables,Copper,H_Scintillator,Lead,M_NEMA_FR4_plate,Silicon,StainlessSteel,WCu, oth; 
-  Air=Cables=Copper=H_Scintillator=Lead=M_NEMA_FR4_plate=Silicon=StainlessSteel=WCu=0.;
+  float Air,Cables,Copper,H_Scintillator,Lead,HGC_G10_FR4,Silicon,StainlessSteel,WCu,oth,Epoxy,Kapton,Aluminium; 
```

Comment:
It is preferred that variable names start with a lowercase letter. Otherwise, they look like class names.